ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-multicast.j2

```diff
@@ -4,6 +4,20 @@
 router multicast
 {%     if router_multicast.ipv4 is arista.avd.defined %}
    ipv4
+{%         if router_multicast.ipv4.rpf.routes is arista.avd.defined %}
+{%             for rpf_route in router_multicast.ipv4.rpf.routes %}
```

Comment:
Need to add a natural_sort filter.

Reply:
Also its required to add a defined test for rpf_route.subnet
My suggestion: ```for rpf_route in router_multicast.ipv4.rpf.routes | arista.avd.natural_sort('subnet') if rpf_route.subnet is arista.avd.defined```

Reply:
Please do the same in documentation templates

Reply:
thanks!

Reply:
The jinja does not like your 1 line suggestion.  I will dig in to this tomorrow unless you have an idea @tgodaA 

Thanks for the assistance!

Reply:
No, the problem was with the artifacts. Did you run molecule converge or molecule test?

Reply:
Yea I caught it just before you typed that... user error.  My input for molecule was incorrect.